astropy/visualization/wcsaxes/core.py

```diff
@@ -195,17 +197,15 @@ def imshow(self, X, *args, **kwargs):
         elif origin == 'upper':
             raise ValueError("Cannot use images with origin='upper' in WCSAxes.")
 
-        # To check whether the image is a PIL image we can check if the data
```

Comment:
This comment looks helpful but is removed. Should we add it back?

Reply:
This comment is obsolete since I'm using `HAS_PIL` to check for PIL.